packages/mui-base/src/Select/Select.test.tsx

```diff
@@ -1006,7 +1006,7 @@ describe('<Select />', () => {
       );
 
       const select = screen.getByRole('combobox');
-      act(() => {
+      await act(async () => {
```

Comment:
These seems out of place 🤔, is it related to the package layout?

Reply:
this test started failing in those PR. correct usage of act fixed it